examples/simplesynth.py

```diff
@@ -11,19 +11,18 @@
 # +
 from typing import Optional
 
-import torch
 import IPython.display as ipd
+import torch
 
+from torchsynth.config import BASE_REPRODUCIBLE_BATCH_SIZE, SynthConfig
 from torchsynth.module import (
     ADSR,
+    VCA,
```

Comment:
This was in alphabetically order?

Reply:
idk I just upgraded to latest black and isort and got these results